src/main/java/picard/illumina/NewIlluminaBasecallsConverter.java

```diff
@@ -74,12 +57,6 @@
      * @param outputRecordClass        Inconveniently needed to create SortingCollections.
      * @param includeNonPfReads        If true, will include ALL reads (including those which do not have PF set)
      * @param ignoreUnexpectedBarcodes If true, will ignore reads whose called barcode is not found in barcodeRecordWriterMap,
-     * @param maxNoCalls               The maximum number of no calls to allow in a match.
-     * @param maxMismatches            The maximum number of mismatched calls to allow in a match.
-     * @param minMismatchDelta         The minimum mismatch difference between the best and second best matches.
-     * @param minimumBaseQuality       The minimum base quality to allow for a matching base call.
-     * @param metrics                  The metrics output for barcode metrics.
-     * @param metricsFile              The metrics file to write out the barcode metrics to.
      */
     public NewIlluminaBasecallsConverter(final File basecallsDir, File barcodesDir, final int lane,
```

Comment:
I thought this class was going to be renamed in a previous review.  Can you rename it now?